drivers/spi/Kconfig.sam

```diff
@@ -11,3 +11,12 @@ config SPI_SAM
 	select GPIO
 	help
 	  Enable support for the SAM SPI driver.
+
+if SPI_SAM
+config SPI_SAM_DMA
+	bool "SPI SAM DMA Support"
+	select DMA
+	help
+	  Enable the SPI DMA mode using XDMAC for SPI instances
+	  that enable dma channels in their device tree node.
```

Comment:
The SPI_SAM_DMA is for all DMA controllers, need to keep text generic. So, you need change the message around `XDMAC`.